test/test_wiki_trac_rst_convert.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+
+from wiki_trac_rst_convert import convert_content
+
+
+class TracRstToVanillaRst(unittest.TestCase):
+    """
+    Test conversion of content from Trac-flavored reStructuredText to
+    vanilla reStructuredText, that is supported by GitHub
+    """
+
+    def assertConvertedContent(self, expected: str, source: str):
+        """
+        Run the Trac RST `source` through the converter,
+        and assert that the output equals `expected`.
+        """
+        self.assertEqual(expected, convert_content(source))
+
+    def test_empty(self):
+        """
+        An empty string is appended a blank line.
+
+        A file not ending in a newline character is not POSIX compliant,
+        and may result in complaints from programs, like `git diff`
+        saying "\ No newline at end of file".
+        https://stackoverflow.com/a/729795/235463
+        """
+        self.assertConvertedContent('\n', '')
+
+    def test_newline(self):
+        """
+        A newline should not be appended another newline.
+        """
+        self.assertConvertedContent('\n', '\n')
+
+    def test_removes_rst_wrapping(self):
+        """
+        The Trac wiki syntax requires reStructuredText to be wrapped in
+        {{{ #!rst }}} markers.
+        They must be removed from the output.
```

Comment:
```suggestion
        It removes the TracWiki RST armor markup from the output.
```
Maybe use just that.

But you docstring is very good :)